Allow to refuse unsolicited responses (#1581)
simplesamlphp/simplesamlphp: docs/simplesamlphp-changelog.md

    to comply with SAML2INT
  * core:PairwiseID and core:SubjectID authprocs no longer support the 'scope' config-setting.
    Use 'scopeAttribute' instead to identify the attribute holding the scope.

>   * Unsolicited responses can denied by disabling it by setting `enable_unsolicited` to `true` in the SP authsource.

Reviewer: ```suggestion
  * Accepting unsolicited responses can be disabled by setting `enable_unsolicited` to `false` in the SP authsource.
```
